app/controllers/home_controller.rb

```diff
@@ -1,2 +1,5 @@
 class HomeController < ApplicationController
+  def hoge
+    puts "hoge"
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.